### ISSUE TITLE
How to convert inline code?

### ISSUE THREAD

**Issue by andrewfan** [kadikraman/draftjs-md-converter#34]
Hey, I'm having a problem converting inline code 

initialValue is \`value\` 

```
          mdToDraftjs(props.value, {
            inlineStyles: {
              Code: {
                type: 'CODE',
                symbol: '`',
              },
            },
          }),
```

Code breaks here code block does not have children
https://github.com/kadikraman/draftjs-md-converter/blob/master/src/mdToDraftjs.js#L175

**Comment by kadikraman** [comment]
Could you check if [this](https://github.com/kadikraman/draftjs-md-converter/pull/37) fixes it?

**Comment by kadikraman** [comment]
Fixed in [v1.1.0](https://github.com/kadikraman/draftjs-md-converter/releases/tag/v1.1.0) 🙏
